server/wfprev-api/src/main/java/ca/bc/gov/nrs/wfprev/data/models/WUIRiskClassRankModel.java

```diff
@@ -17,16 +18,16 @@
 
 @Data
 @EqualsAndHashCode(callSuper = false)
-@JsonRootName(value = "wuiRiskClassCode")
-@Relation(collectionRelation = "wuiRiskClassCode")
+@JsonRootName(value = "wuiRiskClassRank")
+@Relation(collectionRelation = "wuiRiskClassRank")
 @JsonInclude(Include.NON_NULL)
 @Builder
 @AllArgsConstructor
 @NoArgsConstructor
-public class WUIRiskClassCodeModel extends CommonModel<WUIRiskClassCodeModel> {
+public class WUIRiskClassRankModel extends CommonModel<WUIRiskClassRankModel> {
```

Comment:
With the class refactor, do you need to update the graalvm configuration? I didn't see an update to that file (may have just missed it!)

Reply:
Good catch! I just updated graalvm configuration to match the new model and entity name